tests/commonjs.cjs

```diff
@@ -0,0 +1,42 @@
+/**
+ * @fileoverview Tests for checking that the commonjs entry points are still accessible
+ * @author Mike Reinstein
+ */
+
+// eslint-disable-next-line strict
```

Comment:
The `strict` rule knows based on `.eslintrc.cjs` overriding `*.cjs` files' `sourceType` that this file is script source, so it shouldn't complain about this directive.

```suggestion
```